payjoin-cli/src/main.rs

```diff
@@ -136,7 +136,7 @@ fn cli() -> ArgMatches {
     #[cfg(feature = "v2")]
     {
         cmd = cmd.arg(
-            Arg::new("ohttp_relay")
+            Arg::new("ohttp_relays")
```

Comment:
How does this work? Someone passes `--ohttp_relays "https://a.com" --ohttp_relays "https:://b.com"`? Wouldn't we still want the singular `--ohttp_relay` name? Or do you expect them to be passed `--ohttp_relays "https:://b.com","https://b.com` ?

https://docs.rs/clap-action-command/latest/clap_action_command/fn.get_many.html

Reply:
How about we pass them as one or more value like "https://pj.benalleng.com,https://pj.bobspacebkk.com" or just one "https://pj.benalleng.com"?